packages/scratch-vm/src/extensions/scratch3_musiccreation/sheetmusic.js

```diff
@@ -573,6 +579,17 @@ class SheetMusic {
                 x = xinit+xStep;
                 y = y - this.spaceBetween-11*this.staffWidth;
             }
+            if (!(x > 180 && y < 0)) {
+                seen++;
+            } else {
+                signal = signal.slice(seen);
```

Comment:
Hm, I think what I'm confused on is why wouldn't `seen === vizHelper._noteBuf['sheet'].length`?

Reply:
Which in that case, this can be made simpler?

Reply:
Alternative question (that gets at a similar line of thinking)... do you expect this `signal` object to contain any notes?